src/utils/hasCultuurkuurOrganizerLabel.ts

```diff
@@ -0,0 +1,3 @@
+export const hasCultuurkuurOrganizerLabel = (labels: string[]) => {
+  return (labels ?? []).includes('cultuurkuur_organizer');
```

Comment:
I would likely still use a constant but you can export it from this file, because right now it's still in both files consuming the helper (OrganizerForm and [AdditionalInformationStep](https://github.com/cultuurnet/udb3-frontend/blob/feature/III-6565/src/pages/steps/AdditionalInformationStep/AdditionalInformationStep.tsx#L34))